tensorflow/src/TensorFlow/Nodes.hs

```diff
@@ -96,6 +99,21 @@ instance Nodes ControlNode where
 instance a ~ () => Fetchable ControlNode a where
     getFetch _ = return $ pure ()
 
+-- hm, annoying but doable.
+-- unfortunately, pulling out requires a "Identity" everywhere...bleh
```

Comment:
The confusing part is that it's not exactly everywhere. You can try to hide it a bit by composing :| with Identity into a different operator.